Jellyfin.Data/Entities/BaseItemEntity.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Jellyfin.Data.Entities;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning disable CA2227 // Collection properties should be read only
+
+public class BaseItemEntity
+{
+    public required Guid Id { get; set; }
+
+    public required string Type { get; set; }
+
+    public string? Data { get; set; }
+
+    public string? Path { get; set; }
+
+    public DateTime StartDate { get; set; }
+
+    public DateTime EndDate { get; set; }
+
+    public string? ChannelId { get; set; }
+
+    public bool IsMovie { get; set; }
+
+    public float? CommunityRating { get; set; }
+
+    public string? CustomRating { get; set; }
+
+    public int? IndexNumber { get; set; }
+
+    public bool IsLocked { get; set; }
+
+    public string? Name { get; set; }
+
+    public string? OfficialRating { get; set; }
+
+    public string? MediaType { get; set; }
+
+    public string? Overview { get; set; }
+
+    public int? ParentIndexNumber { get; set; }
+
+    public DateTime? PremiereDate { get; set; }
+
+    public int? ProductionYear { get; set; }
+
+    public string? Genres { get; set; }
+
+    public string? SortName { get; set; }
+
+    public string? ForcedSortName { get; set; }
+
+    public long? RunTimeTicks { get; set; }
+
+    public DateTime? DateCreated { get; set; }
+
+    public DateTime? DateModified { get; set; }
+
+    public bool IsSeries { get; set; }
+
+    public string? EpisodeTitle { get; set; }
+
+    public bool IsRepeat { get; set; }
+
+    public string? PreferredMetadataLanguage { get; set; }
+
+    public string? PreferredMetadataCountryCode { get; set; }
+
+    public DateTime? DateLastRefreshed { get; set; }
+
+    public DateTime? DateLastSaved { get; set; }
+
+    public bool IsInMixedFolder { get; set; }
+
+    public string? Studios { get; set; }
+
+    public string? ExternalServiceId { get; set; }
+
+    public string? Tags { get; set; }
+
+    public bool IsFolder { get; set; }
+
+    public int? InheritedParentalRatingValue { get; set; }
+
+    public string? UnratedType { get; set; }
+
+    public float? CriticRating { get; set; }
+
+    public string? CleanName { get; set; }
+
+    public string? PresentationUniqueKey { get; set; }
+
+    public string? OriginalTitle { get; set; }
+
+    public string? PrimaryVersionId { get; set; }
+
+    public DateTime? DateLastMediaAdded { get; set; }
+
+    public string? Album { get; set; }
+
+    public float? LUFS { get; set; }
+
+    public float? NormalizationGain { get; set; }
+
+    public bool IsVirtualItem { get; set; }
+
+    public string? SeriesName { get; set; }
+
+    public string? UserDataKey { get; set; }
```

Comment:
Why this is needed if we have an UserData collection below?

Reply:
Its a double keyed reference. Also imported from legacy data and too much relies on it to just get rid of it would mean changes to the api and clients. 

The collection represents all User's watch data.